std_msgs/Header header
bool is_changing
float32[] is_changing_percentages 
